src/main/java/util/Calculator.java

```diff
@@ -0,0 +1,38 @@
+package util;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+public class Calculator {
+	private static final String CUSTOM_REGEX = "//(.)\n(.*)";
+	private static final String BASIC_REGEX = ",|:";
+	
+	public static int sum(String value) {
+		if (isEmpty(value)) {
+			return 0;
+		}
+		
+		List<Number> list = new ArrayList<>();
+		for (String number: split(value)) {
+			list.add(new Number(number));
+		}	
+		
+		return list.stream().mapToInt(i->i.getNumber()).sum();
```

Comment:
스트림 사용 좋아요! 👍
다음과 같이 메서드 참조를 사용할 수도 있으니 참고해주세요.
```suggestion
		return list.stream().mapToInt(Number::getNumber).sum();
```

Reply:
Number::getNumber 문법 하나 더 배워갑니다. stream 사용법 더 공부하겠습니다.